web/pages/notifications.tsx

```diff
@@ -56,24 +56,38 @@ const HIGHLIGHT_CLASS = 'bg-indigo-50'
 
 export default function Notifications() {
   const privateUser = usePrivateUser()
+  const router = useRouter()
+  const [navigateToSection, setNavigateToSection] = useState<string>()
+  const [activeIndex, setActiveIndex] = useState(0)
 
   useEffect(() => {
     if (privateUser === null) Router.push('/')
   })
 
+  useEffect(() => {
+    const query = { ...router.query }
+    if (query.section) {
+      setNavigateToSection(query.section as string)
+      setActiveIndex(1)
+    }
+  }, [router.query])
+
   return (
     <Page>
       <div className={'px-2 pt-4 sm:px-4 lg:pt-0'}>
         <Title text={'Notifications'} className={'hidden md:block'} />
         <SEO title="Notifications" description="Manifold user notifications" />
 
-        {privateUser && (
+        {privateUser && router.isReady && (
           <div>
-            <Tabs
+            <ControlledTabs
```

Comment:
I actually think it would be nice if the url changed when you clicked into notification settings. (And then you could go back in browser to switch the tab back.)

Did you consider `QueryControlledTabs`?